index.html

```diff
@@ -0,0 +1,75 @@
+<!--
+ * Project: portfolio
+ * File: index.html
+ * Created: 1/12/23
+ * Author: Abdullah Al Mamun <mamun1214@gmail.com>
+-->
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="icon" type="image/x-icon" href="assets/img/icon/favicon.ico">
+    <title>Mamun's Portfolio | Landing Page</title>
+    <link rel="stylesheet" href="assets/css/app.css">
+</head>
+<body>
+<header>
+    <nav class="navbar bg-white">
+        <div class="container">
+            <a id="" href="#" class="navbar-welcome d-flex align-center">
+                <strong>Welcome</strong>
+            </a>
+            <button class="navbar-menu" type="button">
+                <span class="navbar-menu-icon"></span>
+            </button>
+        </div>
+    </nav>
```

Comment:
- [ ] Dear @dev-mamun, Good job making the request changes, but there are some minor details that are missed, the first is that only the Humberger icon need's to be wrapped inside a `nav` tag. I see you wrap everything side a header with a `nav` and that is against the primary requirement. 
- [ ] The other thing is following the changes the number get icon is not visible on the page (despite you adding the background image for the icon). Please make sure your hamburger icon is visible.

![Screenshot 2023-01-13 144326](https://user-images.githubusercontent.com/69077061/212312813-9217dd4d-98cc-49cc-8bf4-787c9551f262.png)